### DISCUSSION TITLE
Name for wrappid extension

### DISCUSSION THREAD

**Discussion by samhere17** [wrappid/.github#18] [Ideas]
Suggest name visible in the marketplace for the wrappid extension.

I suggest **Wrappid Magic**.

Ref: https://github.com/wrappid/wrappidextension/issues/12

@wrappid/core-team

**Comment by anantakumarghosh** [comment]
I suggest **Wrappid Assistant**.

**Comment by Swrnv-qc** [comment]
I would suggest "Wrappid IntelliSense"

**Comment by aitijhyasarkar** [comment]
I would suggest "Wrappid" and its features like intellisense and others can be mentioned in its description.

**Comment by techoneel** [comment]
The final deployed and approved name by the community is wrappid-extension as name of the extension and {vscode}-extensions as repository name.

Closing this discussion since it's discussed and worked upon.
